Resources/config/doctrine/StaticPage.orm.yml

```diff
@@ -0,0 +1,52 @@
+Grossum\StaticPageBundle\Entity\StaticPage:
+    type: entity
```

Comment:
This class should be entity because it has associations and mappedSuperClass can't has it